src/containers/SortingView.tsx

```diff
@@ -181,7 +181,10 @@ const SortingView: React.FunctionComponent<Props> = (props) => {
     return <h3>{`Recording not found: ${sorting.recordingId}`}</h3>
   }
 
-  const selectedUnitIdsLookup: {[key: string]: boolean} = selection.selectedUnitIds.reduce((m, uid) => {m[uid + ''] = true; return m}, {} as {[key: string]: boolean})
+  const sv = plugins.sortingViews['MVSortingView']
+  if (!sv) throw Error('Missing sorting view: MVSortingView')
```

Comment:
We access the MVSortingView through the plugin/extension system, and throw an error if not found.